api/src/lib.rs

```diff
@@ -6,6 +6,15 @@ mod autogenerated_code {
     // Include the auto-generated code.
     include!(concat!(env!("OUT_DIR"), "/protos-auto-gen/mod.rs"));
 }
+
+pub mod tonic {
```

Comment:
I figure push these into a `tonic` namespace to avoid conflicting with the `pub use crate::{autogenerated_code::*` below